docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -1272,16 +1292,16 @@ def get_person_variables(fieldslist, people_vars=generator_constants.PEOPLE_VARS
             people.add(matches.groups()[0])
     else:
       # If it's a PDF name that wasn't transformed by map_names, do one last check
-      # The regex below is non-greedy; _address will match before _mail_address
-      match_pdf_person_suffixes = r"([A-Za-z_]\w*)(" + "|".join(people_suffixes_map.keys()) + "$)"
+      # See regex GitHub issue:
+      # https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/issues/205
+      # TAKE CARE to double check new possible partial matches to suffixes
+      match_pdf_person_suffixes = r"(.+?)(?:(" + "$)|(".join(people_suffixes_map.keys()) + "$))"
```

Comment:
```suggestion
      match_pdf_person_suffixes = r"(.+?)(?:($)|(".join(people_suffixes_map.keys()) + "$))"
```

Also I'm not sure the link to the issue really explains much, but we can better comment later.

Reply:
This was just a wrong suggestion, idk what I was doing. Reverted in https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/commit/e2dcbe239c928a29063b9c30552719a7e3359830.